yarn.lock

```diff
@@ -8250,7 +8250,7 @@ color@^3.0.0:
     color-convert "^1.9.3"
     color-string "^1.6.0"
 
-color@^4.0.1:
+color@^4.0.1, color@^4.2.3:
```

Comment:
those changes in the lock file seem odd and unnecessary. Revert from lock file to state in main?

Reply:
gone ⚔️